__init__.py

```diff
@@ -138,10 +139,27 @@ def handle_glados_intent(self, _):
         path, files = self.get_reference_files("/sounds/glados", "mp3")
         if len(files):
             mp3 = path + "/" + random.choice(files)
-            self.play_audio(mp3)
+            self._play_in_ocp(mp3, title="GlaDOS says...")
         else:
             self.speak_dialog("bad_file")
 
     @skill_api_method
     def get_display_date(self):
         return StarDate().getStardate()
+
+    def _play_in_ocp(self, media, title="Easter Egg!"):
+        data = {
+            "match_confidence": 100,
+            "media_type": 1,  # MediaType.AUDIO
+            "length": 0,
+            "uri": media,
+            "playback": 2,  # PlaybackType.AUDIO
+            "image": "",
+            "bg_image": "",
+            "skill_icon": "",
+            "title": title,
+            "skill_id": self.skill_id,
+        }
+        self.bus.emit(Message(
```

Comment:
consider using https://github.com/OpenVoiceOS/ovos-bus-client/blob/dev/ovos_bus_client/apis/ocp.py#L229

or leave a TODO if you dont want to require the latest alpha of ovos-bus-client

Reply:
@JarbasAl It's been available for several stable versions so I went ahead and added it. Thanks for the callout!